src/AdvancedMH.jl

```diff
@@ -3,6 +3,7 @@ module AdvancedMH
 # Import the relevant libraries.
 using AbstractMCMC
 using Distributions
+using LinearAlgebra
```

Comment:
I'm a fan of more explicit imports (eg to avoid name clashes and make it clearer which package owns objects and functions) - could we change this to
```suggestion
import LinearAlgebra
```
or
```suggestion
using LinearAlgebra: I
```
?